src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -943,6 +948,11 @@ define([
 
                     tabs: {
                         url: navigationUrl,
+                        options: {
+                            data: function() {
+                                return this.sandbox.util.extend(false, {}, this.content.toJSON());
```

Comment:
tbh i have copied this line from the contact. but i will change it to `deepCopy`.